src/models/EET.ts

```diff
@@ -6,13 +6,14 @@
  * @year 2022
  */
 
-import axios from "axios";
-import { GoPay } from "..";
-import { eet } from "../types/eet";
+import { GoPay } from "~/goPay";
+import { eet } from "~/types/eet";
 
 export class Misc {
+  // @ts-ignore
```

Comment:
Why are you putting `// @ts-ignore`?

Reply:
The IDE was telling me that these properties weren’t used anywhere and were throwing me red underline, which was driving me crazy 😁

Reply:
Ah okie then 👍🏻